apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/LiveAppSDKLogic.ts

```diff
@@ -48,7 +50,12 @@ export const requestAccountLogic = async (
    */
   const safeCurrencies = currencies?.filter(c => typeof c === "string") ?? undefined;
```

Comment:
Nothing for you, just a note for me as I just discovered this line in our code and it's pretty inefficient to do this here